src/strings/index.tsx

```diff
@@ -562,6 +567,11 @@ const strings = new LocalizedStrings({
     DELETE_ACCESSION_MESSAGE: 'You’re about to delete accession {0}.',
     ARE_YOU_SURE: 'Are you sure?',
     LATITUDE_LONGITUDE: 'Latitude, Longitude',
+    WITHDRAW: 'Withdraw',
+    AMOUNT_REMAINING: 'Amount ({0} remaining)',
+    WITHDRAW_ALL: 'Withdraw all',
+    CT: 'ct',
```

Comment:
i have a feeling this was not intentional, we should probably spell out 'count'..

Reply:
I asked Ming in figm